css/contacts.scss

```diff
@@ -50,6 +50,7 @@ $grid-input-height-with-margin: $grid-height-unit - $grid-input-margin * 2;
 
 .app-content-details {
 	padding: 0 44px 80px 44px;
+	overflow: auto;
```

Comment:
Maybe this should go into the vue components?

Reply:
I am not sure. For now this is the only app that needs this but that's your call.

Reply:
Yeah, no, not every app uses virtual scrolling